specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -168,6 +173,7 @@ export function TreeRow({
                     ? 'outline outline-1 outline-blue-500'
                     : ''
                 }
+                ${hideEmptyNodes && isLoadingStats ? 'opacity-50' : ''}
```

Comment:
this solution is good by me, if user-facing people approve. Can you ask Grant or others?